docs/cn/deploy/Running-Alluxio-On-Kubernetes.md

```diff
@@ -1,128 +1,1112 @@
+
+{% endcollapsible %}
+
+{% collapsible Example: HDFS as the under store %}
+首先为HDFS客户端所需的任何配置创建secrets。它们将挂载在`/secrets`下。
+
 ```console
-$ cp conf/alluxio.properties.template conf/alluxio.properties
+$ kubectl create secret generic alluxio-hdfs-config --from-file=${HADOOP_CONF_DIR}/core-site.xml --from-file=${HADOOP_CONF_DIR}/hdfs-site.xml
+```
+
+```properties
+properties:
+  alluxio.master.mount.table.root.ufs: "hdfs://<ns>"
+  alluxio.master.mount.table.root.option.alluxio.underfs.hdfs.configuration: "/secrets/hdfsConfig/core-site.xml:/secrets/hdfsConfig/hdfs-site.xml"
+secrets:
+  master:
+    alluxio-hdfs-config: hdfsConfig
+  worker:
+    alluxio-hdfs-config: hdfsConfig
+```
+
+
+
+{% endcollapsible %}
+
+{% collapsible Example: Off-heap Metastore Management in Persistent Volumes %} 
+以下配置为每个Alluxio master Pod创建一个`PersistentVolumeClaim`，并将Pod
+配置为使用该卷作为一个on-disk基于RocksDB的metastore。
+
+```properties
+properties:
+  alluxio.master.metastore: ROCKS
+  alluxio.master.metastore.dir: /metastore
+
+metastore:
+  volumeType: persistentVolumeClaim # Options: "persistentVolumeClaim" or "emptyDir"
+  size: 1Gi
+  mountPath: /metastore
+  # Attributes to use when the metastore is persistentVolumeClaim
+  storageClass: "standard"
+  accessModes:
+   - ReadWriteOnce
+```
+
+  
+{% endcollapsible %}
+
+{% collapsible Example: Off-heap Metastore Management in `emptyDir` Volumes %} 
+以下配置为每个Alluxio master Pod创建一个`emptyDir`卷，并将Pod配置为使用该卷作为一个on-disk基于RocksDB的metastore。
+
+```properties
+properties:
+  alluxio.master.metastore: ROCKS
+  alluxio.master.metastore.dir: /metastore
+
+metastore:
+  volumeType: emptyDir # Options: "persistentVolumeClaim" or "emptyDir"
+  size: 1Gi
+  mountPath: /metastore
+  # Attributes to use when the metastore is emptyDir
+  medium: ""
 ```
 
+
+>注意：`emptyDir`卷的寿命与Pod寿命相同。 
+它不是持久性存储。
+当Pod重新启动或被重新调度时，Alluxio日志将丢失。请仅在实验用例中使用。检查[emptyDir](https://kubernetes.io/docs/concepts/storage/volumes/#emptydir)了解更详细信息)。
+
+  {％endcollapsible％}
+
+{%collapsible Example: Multiple Secrets %} 
+ 可以将多个secrets同时挂载到master和workerPods。每个Pod区域的格式为`<secretName>：<mountPath>`
+
+```properties
+secrets:
+  master:
+    alluxio-hdfs-config: hdfsConfig
+    alluxio-ceph-config: cephConfig
+  worker:
+    alluxio-hdfs-config: hdfsConfig
+    alluxio-ceph-config: cephConfig
+```
+
+
+{％endcollapsible％}
+
+{% collapsible Examples: Alluxio Storage Management %} 
+Alluxio在worker Pods上管理本地存储，包括内存。[Multiple-Tier Storage]({{ '/en/core-services/Caching.html#multiple-tier-storage' | relativize_url }}) 可以使用以下参考配置来设置。
+
+支持的3 种卷类型 `type`：[hostPath](https://kubernetes.io/docs/concepts/storage/volumes/#hostpath)，[emptyDir](https://kubernetes.io/docs/concepts/storage/volumes/＃emptydir) 
+ 和[persistentVolumeClaim](https://kubernetes.io/docs/concepts/storage/volumes/#persistentvolumeclaim)。
+
+**仅内存级**
+
+```properties
+tieredstore:
+  levels:
+  - level: 0
+    mediumtype: MEM
+    path: /dev/shm
+    type: emptyDir
+    high: 0.95
+    low: 0.7
+```
+
+
```

Comment:
Alluxio doc convention is not having too empty lines together. Delete one of them, thanks
Please apply this to all following docs which have two consecutive empty lines